src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1476,6 +1486,9 @@ public static async Task<Stream> ReadAsStreamUncompressedAsync(HttpResponseMessa
                 return false;
             }
 
+            // As the server is down, we discard the server version to ensure we update when it goes up. 
+            chosenNode.DiscardServerVersion();
```

Comment:
I am not entirely sure this is the proper place to discard the Server Version on failure.

Reply:
Looks like it should do the trick, yes.